src/twisted/conch/scripts/conch.py

```diff
@@ -313,10 +313,6 @@ def serviceStarted(self):
         conn = self
         self.localForwards = []
         self.remoteForwards = {}
-        if not isinstance(self, connection.SSHConnection):
-            # make these fall through
-            del self.__class__.requestRemoteForwarding
-            del self.__class__.cancelRemoteForwarding
```

Comment:
1- I wonder why this code is here, since this is a method of `SSHConnection`, which is itself a subclass of `connection.SSHConnection`, and `self` is the first argument to the method, so it's unclear how it was possible for `self` not to be an instance of `connection.SSHConnection`.

2- Whatever this was meant to do, it was untested, since it's removal didn't break any tests.

Someone familiar with `twisted.conch` should have a loo at this, I guess.

Reply:
FWIW. That code was added 17 years ago by `z3p`. That's a pre-GitHub user…